scripts/stack/README.md

```diff
@@ -0,0 +1,108 @@
+# Full Elastic Stack with Connectors
+
+We know that setting up the Connectors service may look a bit daunting, but to help with this, we pulled together a set of scripts to enable a user to get up and running with a full stack with connectors. Here, we provide an example configuration, Docker Compose configuration, and a bash script to startup and run a stack including Elasticsearch, Kibana, and Connectors.
+
+These scripts can be useful if you want to try out Connectors and ingesting their data into an Elasticsearch instance, or, as a starting place for adding Connectors to your production environment.
+
+# Contents
+
+* [Prerequisites](#prerequisites)
+* [Running the Stack](#running-the-stack)
+* [Prompted Configuration](#prompted-configuration)
+* [Manual Configuration](#manual-configuration)
+* [Script command reference](#script-command-reference)
+  * [run-stack.sh](#run-stacksh)
+  * [stop-stack.sh](#stop-stacksh)
+
+## Prerequisites
+* Linux or MacOS (Although Windows can run the Connectors service, it is not currently supported via these scripts)
+* Docker with Docker Compose Installed
+* It is recommended to run Docker with at least 4GB of available RAM.
+
+## Running the Stack
+
+If you do not wish to use the default Elasticsearch username and password, you must set the username and password (or the API key)
+in the default [config.yml](./config.yml) file before running the `run-stack.sh` script. You can also
+use a different password for the `elastic` user by specifying the environment variable
+`ELASTIC_PASSWORD` before running the script. E.g.:
+
+```bash
+$ ELASTIC_PASSWORD="my_new_password" ./scripts/stack/run-stack.sh
+```
+
+## Prompted Configuration
+
+If you run the `run-stack.sh` command without any flags, by default the script will
+ask if you want to set up the connectors configuration. If you enter "y" to run
+the configurator, it will take you through a set of prompts using the
+[Connectors CLI](../../connectors/connectors_cli.py) to create a new index and connector.
+
+The resulting configuration will be saved to the [config.yml](./connectors-config/config.yml)
+file that is used to start the Connectors container. Note that using the configurator,
+it will modify the [connectors config.yml](./connectors-config/config.yml) file. You can restore it
+by copying over the [example config.yml.example](./connectors-config/config.yml.example) file
+to `config.yml` in the directory.
+
+If you configured a new connector, you will need to go into Kibana to start or schedule a sync.
```

Comment:
I think CLI is designed to be an alternative to Kibana, and we can document how to schedule/trigger a sync via Connectors CLI, and make Kibana optional (e.g. only start Kibana if `-k` is specified)

Reply:
True, but these scripts are meant to provide a quick gateway to getting a Connectors framework up and running, and thus, the usage of the CLI is very "lite" here and only used for configuration - but the instructions in this particular part is for if the user want to manually set things up... I'll add in some extra documentation though around the CLI if the user wants to take that route.